.github/workflows/e2e-manual-icad.yaml

```diff
@@ -25,7 +25,7 @@ on:
           - master
           - v0.3.2
           - v0.2.2
-          - v0.1.2
+          - v0.1.3
```

Comment:
tests should pass with this version once [this pr](https://github.com/cosmos/ibc-go/pull/2036) is merged

Reply:
Merged!